### DISCUSSION TITLE
[Prettier Plugin] tailwindcss plugin

### DISCUSSION THREAD

**Discussion by yzhuuuu** [tailwindlabs/tailwindcss#12150] [Help]
Hi there,
I followed the readme and created the prettier.config.js file. However, I got some error like 
`prettier.config.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename prettier.config.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /Users/yyyy-/Desktop/receipt-app/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).`

here is the package.json
`{
  "name": "receipt-app",
  "private": true,
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "vite build",
    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
    "preview": "vite preview"
  },
  "dependencies": {
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-icons": "^4.11.0",
    "react-router-dom": "^6.16.0"
  },
  "devDependencies": {
    "@types/react": "^18.2.15",
    "@types/react-dom": "^18.2.7",
    "@vitejs/plugin-react": "^4.0.3",
    "autoprefixer": "^10.4.16",
    "eslint": "^8.45.0",
    "eslint-plugin-react": "^7.32.2",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-react-refresh": "^0.4.3",
    "postcss": "^8.4.31",
    "prettier": "^3.0.3",
    "prettier-plugin-tailwindcss": "^0.5.5",
    "tailwindcss": "^3.3.3",
    "vite": "^4.4.5"
  }
}
`

**Comment by s0h311** [comment]
Try using a default export in your `tailwind.config.js`.

```
const tailwindConfigs = {
  // your configs
}

export default tailwindConfigs
```

and also make sure your postcss config file is named `postcss.config.cjs` and looks like this

```
module.exports = {
  // your configs or just the default ones
}
```
